src/resolvers/admin.ts

```diff
@@ -23,4 +24,20 @@ export class AdminResolver {
     await killSwitch.save()
     return { isKilled }
   }
+
+  @Mutation(() => GeneratedSignature)
+  async generateSignature(@Arg('bytes', () => String) bytes: string) {
```

Comment:
We will most likely need to do some additional validation here, see https://github.com/Joystream/joystream/issues/4307#issuecomment-1361311240